Charts.xcodeproj/project.pbxproj

```diff
@@ -1050,7 +1058,11 @@
 				INFOPLIST_FILE = "Source/Supporting Files/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/Frameworks";
+				LD_RUNPATH_SEARCH_PATHS = (
```

Comment:
@jjatie are these diffs generated by Xcode?

Reply:
Yea, those are generated by Xcode. Looks like the file format used to use a space separated string, and now it uses something like a tuple